### DISCUSSION TITLE
external-html for consent?

### DISCUSSION THREAD

**Discussion by leighgreenberg** [jspsych/jsPsych#2355] [Q&A]
I've created an html consent form that I'd like to add to my experiment. I added the code as it appears on https://www.jspsych.org/7.0/plugins/external-html/ and ensured the variable names match. I've also added a line to load the plugin, but I still can't get the page to display. Both the consent.html and the original experiment (i.e. without consent) load just fine on their own. Is there something about this plugin that isn't listed on the jspsych website?

**Comment by jodeleeuw** [comment]
Hi @leighgreenberg,

Is there a link to your experiment that we could look at? If not, are there any error message in the browser console that you see?

**Reply by leighgreenberg** [reply]
Hi @jodeleeuw, I've mostly been running locally but I've just updated the online version, here: https://gitlab.pavlovia.org/greenbee/ci-experiment

**Comment by leighgreenberg** [comment]
Update: turns out the problem was that I was running locally.. I kept getting in browser errors that went away once I switched type from jsPsychExternalHtml to "external-html" and running it on Pavlovia.
